mojo/public/cpp/platform/tcp_platform_handle_utils_posix.cc

```diff
@@ -88,7 +88,7 @@ base::ScopedFD CreateTCPClientHandle(size_t port) {
   return handle;
 }
 
-base::ScopedFD CreateTCPServerHandle(size_t port) {
+base::ScopedFD CreateTCPServerHandle(uint16_t port, uint16_t* out_port) {
```

Comment:
It might be better to use only one pointer parameter (uint16_t* port).
CHECK(port)
if (*port == 0)
// extract a port number and then
// *port = new_port;
Of course we'll have to modify the code that calls this function.

Reply:
We can not use constant directly. I think adding the return parameter is more simple.
Before :
mojo::CreateTCPServerHandle(mojo::kCastanetsNonBrokerPort)
After :
uint16_t port = mojo::kCastanetsNonBrokerPort;
mojo::CreateTCPServerHandle(&port)